projects/1.21/assets/decostorage/deco_storage/lang/zh_cn.json

```
@@ -0,0 +1,91 @@
+{
+  "creativetab.deco_storage.mod_tab": "装饰储物",
+
+  "block.deco_storage.storage_crafter": "贮藏合成器",

```

```suggestion
  "block.deco_storage.storage_crafter": "储物合成器",
```
仅建议，可改可不改
其实没有啥特别的理由，主要是为与模组名统一，以及storage译作贮藏之前没怎么见过